apps/root/src/common/components/token-selector/index.tsx

```diff
@@ -150,7 +150,7 @@ const TokenSelector = ({ handleChange, selectedToken }: TokenSelectorProps) => {
       parsedWithCustomTokens,
       [
         ({ balanceUsd, token }) =>
-          formatUsdAmount({ amount: formatUnits(balanceUsd || 0n, token.decimals + 18), intl }),
+          Number(formatUsdAmount({ amount: formatUnits(balanceUsd || 0n, token.decimals + 18), intl })),
```

Comment:
This can fail if number is comma separated => Number("1,1") = NaN

Reply:
Oh really good carch